packages/create-plugin/src/utils/utils.handlebars.ts

```diff
@@ -25,7 +25,9 @@ export const normalizeId = (pluginName: string, orgName: string, type: PLUGIN_TY
 
   const newPluginName = pluginName.replace(re, '').replace(nameRegex, '');
   const newOrgName = orgName.replace(nameRegex, '');
-  return newOrgName.toLowerCase() + '-' + newPluginName.toLowerCase() + `-${type}`;
+  const newType = type === PLUGIN_TYPES.scenes ? PLUGIN_TYPES.app : type;
```

Comment:
Nice! (Maybe it would be nice in the future to define an explicit mapping between these two different "types", e.g. in the constants.)